.github/workflows/ci.yml

```diff
@@ -55,7 +60,16 @@ jobs:
           python -m pip install cumulusci
 
       - name: Run Tests
-        run: python -m pytest
+        run: |
+          python -m coverage run -m pytest
+
+      - name: Check Coverage
+        run: |
+          coverage xml
+          coverage report
+          git branch -a
+          diff-cover coverage.xml --fail-under 100 --compare-branch=origin/main --diff-range-notation=.. --show-uncovered --markdown-report coverage.md
+          cat coverage.md >> $GITHUB_STEP_SUMMARY
```

Comment:
This also works:
```suggestion
          diff-cover coverage.xml --fail-under 100 --compare-branch=origin/main --diff-range-notation=.. --show-uncovered --markdown-report $GITHUB_STEP_SUMMARY
```

Reply:
@jstvz I did it in two steps in case some previous step writes to GITHUB_STEP_SUMMARY at some point. `>>` appends and `-markdown-report $GITHUB_STEP_SUMMARY` overwrites

Reply:
Each step (i.e., `- run:`) has it's own step summary. All steps are concatenated into the final output.